src/current/cockroachcloud/physical-cluster-replication.md

```diff
@@ -53,28 +53,71 @@ You'll need the following:
     We recommend creating service accounts with the [principle of least privilege](https://wikipedia.org/wiki/Principle_of_least_privilege), and giving each application that accesses the API its own service account and API key. This allows fine-grained access to the cluster and PCR streams.
     {{site.data.alerts.end}}
 
-For the schema of each API response, refer to the [CockroachDB Cloud API reference documentation](https://www.cockroachlabs.com/docs/api/cloud/v1/physical-replication-streams).
+For the schema of each API response, refer to the [CockroachDB Cloud API reference documentation](https://www.cockroachlabs.com/docs/api/cloud/v1.html#get-/api/v1/physical-replication-streams).
 
 ### Step 1. Create the clusters
 
 To use PCR, it is necessary to set the `supports_cluster_virtualization` field to `true`. This setting enables cluster virtualization, which is the architecture that supports PCR. For details on supported cluster cloud provider and region setup, refer to the [prerequisites section](#before-you-begin).
 
-1. Send a `POST` request to create the primary cluster:
+1. Send a `POST` [request](https://www.cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/physical-replication-streams) to create the primary cluster:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    curl --location --request POST 'https://cockroachlabs.cloud/api/v1/clusters' --header "Authorization: Bearer api_secret_key" --header 'Content-Type: application/json' --data '{"name": "primary_cluster_name", "provider": "AWS", "spec": {"dedicated": {"cockroachVersion": "v25.2", "hardware": {"disk_iops": 0, "machine_spec": {"num_virtual_cpus": 4}, "storage_gib": 16}, "region_nodes": {"us-east-1": 3}, "supports_cluster_virtualization": true}}}'
+    curl --location --request POST 'https://cockroachlabs.cloud/api/v1/clusters' \
+    --header "Authorization: Bearer {api_secret_key}" \
+    --header 'Content-Type: application/json' \
+    --data '{
+      "name": "{primary_cluster_name}", 
+      "provider": "AWS", 
+      "spec": {
+        "dedicated": {
+          "cockroachVersion": "v25.2", 
+          "hardware": {
+            "disk_iops": 0 "machine_spec": {"num_virtual_cpus": 4}, 
```

Comment:
Still a couple lines that need to be properly formatted, see https://github.com/cockroachdb/docs/blob/9487bfb5f4602a9d17e6ba785f9787893dbc37b9/src/current/cockroachcloud/cloud-api.md?plain=1#L368 for an example